ios/Podfile.lock

```diff
@@ -6,10 +6,7 @@ PODS:
     - JWT (~> 3.0.0-beta.7)
```

Comment:
Also bit suspicious; doesn't match what is on 5.1.0

Reply:
Oops, my local base branch was outdated. Updated